test/drop_infra.bats

```diff
@@ -16,18 +16,18 @@ function teardown() {
 	cleanup_test
 }
 
-@test "test infra ctr dropped" {
-	python -c 'import json,sys;obj=json.load(sys.stdin);obj["linux"]["security_context"]["namespace_options"]["pid"] = 1; json.dump(obj, sys.stdout)' \
-		< "$TESTDATA"/sandbox_config.json > "$TESTDIR"/sandbox_no_infra.json
+@test "test infra ctr not dropped" {
```

Comment:
Looks like you have renamed the test, which resulted in a collision:

Error: Duplicate test name(s) in file "/home/circleci/go/src/github.com/cri-o/cri-o/test/./drop_infra.bats": test_test_infra_ctr_not_dropped